### PR TITLE
feat(); Pancake V3; Add Pancake V3 BSC/Ethereum subgraphs

### PR DESCRIPTION
**Pull request by steegecs**
**Context:**
Uniswap V3 forks are now available, and Pancake V3 just released a Uniswap V3 fork. The contract code is almost identical with a few small additions for liquidity mining. This PR integrates the Pancake V3 subgraph in with the Uniswap V3 subgraphs.

### CODE REVIEW

File: subgraphs/uniswap-v3/src/common/price/price.ts [hunk 1]
```diff
@@ -62,6 +62,12 @@ export function sqrtPriceX96ToTokenPrices(
 export function getNativeTokenPriceInUSD(nativeToken: Token): BigDecimal {
   let stableAmount = BIGDECIMAL_ZERO;
   let tokenIndicator: i32;
+
+  if (NetworkConfigs.getStableOraclePools().length == 0) {
+    log.warning("No STABLE_ORACLE_POOLS given", []);
```
[the remark this answers is not in the document]
Comment (steegecs) [reply subgraphs/uniswap-v3/src/common/price/price.ts]: No - I removed this. I put this here for a reason when I was testing, but not necessary.

**Comment by dmelotik** [comment]
@steegecs do you have a test deployment?

File: package.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
   "dependencies": {
+    "messari-subgraph-cli": "^0.1.52",
```
Comment (dmelotik) [inline package.json:3]: Asking for a friend, why would we want this in the package.json?

**Review by dmelotik — APPROVED** [review]
LGTM! Should we name the top level directory `uniswap-v3-forks`?

File: package.json (hunk 2 above)
Comment (steegecs) [reply package.json:3]: Don't need it!

**Pull request merged**